compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmMethodGen.java

```diff
@@ -752,6 +755,10 @@ private static boolean isModuleTestInitFunction(BIRPackage module, BIRFunction f
         return func.name.value.equals(calculateModuleSpecialFuncName(packageToModuleId(module), "<testinit>"));
     }
 
+    private static boolean isModuleStartFunction(BIRPackage module, BIRFunction function) {
+        return function == module.functions.get(1);
+    }
```

Comment:
Shall we have the called function after the calling method? This improves vertical readability